utils/jira.js

```diff
@@ -26,9 +20,7 @@ const jira = new JiraClient({
 })
 
 const fetchHeader = {
-  Authorization: `Basic ${Buffer.from(
-    `${options.username}:${options.token}`
-  ).toString('base64')}`,
+  Authorization: `Basic ${Buffer.from(`${options.username}:${options.token}`).toString('base64')}`,
```

Comment:
I've never used `toString('base64')` to encode a string. Usually use the [btoa](https://developer.mozilla.org/en-US/docs/Web/API/btoa) function.

```js
btoa(`${options.username}:${options.token}`)
```

Reply:
> I've never used `toString('base64')` to encode a string. Usually use the [btoa](https://developer.mozilla.org/en-US/docs/Web/API/btoa) function.

> ```js
> btoa(`${options.username}:${options.token}`)
> ```
>

When starting to implement this suggestion, the jsdoc in my ide informed me:

```
btoa(data: string): string
An ASCII (Latin1) string.

Decodes a string into bytes using Latin-1 (ISO-8859), and encodes those bytes into a string using Base64.
The data may be any JavaScript-value that can be coerced into a string.

This function is only provided for compatibility with legacy web platform APIs and should never be 
used in new code, because they use strings to represent binary data and predate the introduction 
of typed arrays in JavaScript. For code running using Node.js APIs, converting between 
base64-encoded strings and binary data should be >performed using Buffer.from(str, 'base64') 
and buf.toString('base64').

@since — v15.13.0, v14.17.0
@deprecated — Use buf.toString('base64') instead."
```